generate/write-to-json-file.js

```diff
@@ -0,0 +1,45 @@
+import path from 'path'
+import fs from 'fs'
+
+const JsonStringifyParameters = {
+  replacer: null,
+  space: 4,
+}
+
+export function writeJsonArray(arrayNewItems, config, applyFilter = false) {
+  const DEST_JSON_FILE = path.join(__dirname, config.destDirectoryFromHere, `${config.destFileName}.json`)
```

Comment:
It is your repo (your baby) so I just follow what tell me without hesitation.
`data.json` is now hard coded and removed from configs.